### PR TITLE
Create recipe route and simple display for queries

### LINKED ISSUE #2: Create recipe index page route
- We need a page to live at `/recipes`
- This requires a new `recipes.tsx` component in the `/pages` directory
- This should also query all the recipes and render them in some super simple way

### PR DESCRIPTION
**Pull request by its-edmund**
- Added `recipes.tsx` in `/pages` directory
- Added basic display for data queries
Screen Shot 2021-02-12 at 3 17 11 PM

### CODE REVIEW

File: src/pages/recipes.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+import styles from '../css/Blog.module.css';
+
+interface Props extends PageProps {
+  data: GatsbyTypes.RecipeQueryQuery;
+}
+
+function RecipesIndex(props: Props): JSX.Element {
+  const siteTitle = props.data.site?.siteMetadata?.title;
+  const posts = props.data?.allContentfulRecipe?.edges;
```
Comment (migueldlr) [inline src/pages/recipes.tsx:16]: nit: would prefer `recipes` rather than `posts`

File: src/pages/recipes.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        <div className={styles.hero}>Recipes</div>
+        <div className="wrapper">
+          <h2 className="section-headline">Wonderful Recipes</h2>
+          <Divider />
+          <ul className="article-list">
+            {posts.map(({ node }) => {
+              return (
+                <li key={node.id}>
+                  <h1>{node.title}</h1>{' '}
+                  <h1>{`${String(node.prepTime)} ${node.prepTime === 1 ? 'min' : 'mins'}`}</h1>
```
Comment (migueldlr) [inline src/pages/recipes.tsx:31]: nit: could just be `${node.prepTime}`

File: src/pages/recipes.tsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+export const pageQuery = graphql`
+  query RecipeQuery {
+    site {
+      siteMetadata {
+        title
+      }
+    }
+    allContentfulRecipe {
+      edges {
+        node {
```
Comment (migueldlr) [inline src/pages/recipes.tsx:54]: Is there a specific reason we query `edges.node` rather than just `nodes`?

File: src/pages/recipes.tsx [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+        <div className="wrapper">
+          <h2 className="section-headline">Wonderful Recipes</h2>
+          <Divider />
+          <ul className="article-list">
+            {posts.map(({ node }) => {
+              return (
+                <li key={node.id}>
+                  <h1>{node.title}</h1>{' '}
+                  <h1>{`${String(node.prepTime)} ${node.prepTime === 1 ? 'min' : 'mins'}`}</h1>
+                  <h1>{`${String(node.totalTime)} ${node.totalTime === 1 ? 'min' : 'mins'}`}</h1>
```
Comment (migueldlr) [inline src/pages/recipes.tsx:32]: ditto: `${node.totalTime}`

**Review by migueldlr — APPROVED** [review]
Looks great! Good work! 🥳

File: src/pages/recipes.tsx (hunk 1 above)
Comment (its-edmund) [reply src/pages/recipes.tsx:16]: gotcha, will change it for future

File: src/pages/recipes.tsx (hunk 2 above)
Comment (its-edmund) [reply src/pages/recipes.tsx:31]: eslint was giving an error for some reason but this seemed to fix it

**Pull request merged**